src/main/java/scm/address/model/filename/Filename.java

```diff
@@ -7,10 +7,10 @@
  * Guarantees: immutable; filename is valid as declared in {@link #isValidFilename(String)}
  */
 public class Filename {
-    public static final String MESSAGE_CONSTRAINTS = "Filenames should be alphanumeric.";
+    public static final String MESSAGE_CONSTRAINTS = "Filenames should be alphanumeric and json/csv files.";
```

Comment:
`json/csv files` should be `.json or .csv files` for clarity, I think?

Reply:
That would be better! Changing!